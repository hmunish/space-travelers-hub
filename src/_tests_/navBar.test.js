import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/navBar';

describe('test navBar component', () => {
    it('should render correct elements', () => {
        const { getByTestId } = render(
            <BrowserRouter>
              <Navbar />
            </BrowserRouter>,
          );
          const bar = getByTestId('navi');
          expect(bar).toBeInTheDocument;

    });

    it('should render correct snapshot', () => {
        const tree = renderer.create(
            <BrowserRouter>
              <Navbar />
            </BrowserRouter>,
          ).toJSON;
         
          expect(tree).toMatchSnapshot();

    });
});