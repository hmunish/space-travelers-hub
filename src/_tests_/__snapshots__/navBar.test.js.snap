// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navBar component should render correct snapshot 1`] = `[Function]`;
